linera-rpc/src/message.rs

```diff
@@ -45,8 +45,8 @@ pub enum RpcMessage {
     GenesisConfigHashResponse(Box<CryptoHash>),
     DownloadBlobContentResponse(Box<BlobContent>),
     DownloadConfirmedBlockResponse(Box<ConfirmedBlock>),
-    DownloadCertificateResponse(Box<Certificate>),
-    DownloadCertificatesResponse(Box<Vec<Certificate>>),
+    DownloadCertificateResponse(Box<ConfirmedBlockCertificate>),
```

Comment:
(Unrelated but: I wonder if we still need the singular version; not sure if we use it, and if we do, we can request `vec![hash]` instead.)

Reply:
I wanted to do that in a separeate PR but this one is so small that I'm gonna do it here.

Reply:
Done in [RM singular DownloadCertificate RPC query.](https://github.com/linera-io/linera-protocol/pull/2976/commits/2d41fa9789feef9f50eb5188a2aa4e7078dfb038)